version.yml

```diff
@@ -1,4 +1,4 @@
-package: 1.20.12-1
+package: 1.21.7
```

Comment:
This is a bit of a problem. Dropping the suffix here means we can't fix an error in this package, for example a mistake when packaging it.

We need to think of something else.

Reply:
Oki, i readded the `-1` for the iteration.